### PR TITLE
Use logger instead of print in qa

### PR DESCRIPTION
**Pull request by janisz**
## Description

This PR make use of logger in all tests.
It also prevent use of print functions.
This is follow up of work done in #1387 

## Splitted into following PRs

- [x] #1734 
- [ ] #1736 
- [x] #1737 
- [ ] https://github.com/stackrox/stackrox/pull/1760

## Testing Performed

CI

### CODE REVIEW

**Comment by gavin-stackrox** [comment]
If BAT tests pass please retest with `all` and over a larger set of cluster flavors (being careful not to exacerbate any existing OSD quota problems i.e. not aro/rosa/osd yes on openshift-4, aks, eks).

**Review by porridge — APPROVED** [review]
AWESOME!

File: qa-tests-backend/src/test/groovy/AdmissionControllerNoImageScanTest.groovy [hunk 1]
```diff
@@ -124,11 +123,11 @@ class AdmissionControllerNoImageScanTest extends BaseSpecification {
                     orchestrator.deleteDeployment(deployment)
                     deleted = true
                 } catch (NullPointerException ignore) {
-                    println "Caught NPE while deleting deployment, retrying in 1s..."
+                    log.info "Caught NPE while deleting deployment, retrying in 1s..."
                 }
             }
             if (!deleted) {
-                println "Warning: failed to delete deployment. Subsequent tests may be affected ..."
+                log.warn "failed to delete deployment. Subsequent tests may be affected ..."
```
Comment (md2119) [inline qa-tests-backend/src/test/groovy/AdmissionControllerNoImageScanTest.groovy:130]: Prior to this change, I believe most, if not all log messages were sentence case. We should make sure that there is consistency in the new way, sentence case or no sentence case. Now, the above info is a sentence case and this warning is not.

**Review by md2119 — APPROVED** [review]

**Comment by janisz** [comment]
I think I'll split this PR into bunch of smaller

**Comment by janisz** [comment]
Closing as this was splitted into multiple PRs

**Pull request closed without merging**